ui/frontend/build_src/src/components/creationPanel/makeButton/index.tsx

```diff
@@ -1,24 +1,60 @@
 import React, {useEffect, useState}from "react";
 
 import { useImageCreate } from "../../../store/imageCreateStore";
-// import { useImageDisplay } from "../../../store/imageDisplayStore";
 import { useImageQueue } from "../../../store/imageQueueStore"; 
-// import { doMakeImage } from "../../../api"; 
 import {v4 as uuidv4} from 'uuid';
 
+import { useRandomSeed } from "../../../utils";
+
 export default function MakeButton() {
 
+  const parallelCount = useImageCreate((state) => state.parallelCount);
   const builtRequest = useImageCreate((state) => state.builtRequest);
   const addNewImage = useImageQueue((state) => state.addNewImage);
   
-  const makeImage = () => {
-    // todo turn this into a loop and adjust the parallel count
-    // 
-    const req =  builtRequest();
-    addNewImage(uuidv4(), req)
+  const makeImages = () => {
+    const req = builtRequest();
+    let requests = [];
```

Comment:
this store the actual number of requests, and out my outputs each one will want.